app/dashboard/models.py

```diff
@@ -2121,6 +2142,8 @@ class Profile(SuperModel):
     )
     keywords = ArrayField(models.CharField(max_length=200), blank=True, default=list)
     organizations = ArrayField(models.CharField(max_length=200), blank=True, default=list)
+    orgs = models.ManyToManyField(Organization, blank=True)
```

Comment:
should this have a more descriptive name to avoid confusions with "organizations" ? Maybe something more descriptive as orgs_owner or orgs_perm. Because there is so many orgs organizations already on the app.

Reply:
will do

Reply:
Ideally this would be `organizations = models.ManyToManyField(Organization, blank=True)` and replace the current organizations column value with a list from the Organization model.

There is a reference to `profile.organizations` in one of the popover javascript templates, yet upon further inspection it seems that the the `.organizations` variable is not set from the Profile.organizations model data that we're changing here.

The reason I did this was to revert that change, though I couldn't find any breaking changes as a result of the switch, it doesn't mean there aren't any.

I purpose profile_organizations.